Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml

```diff
@@ -1,76 +1,76 @@
 <Document>
-# Airlock Upgrades
-It is not uncommon for plucky individuals to try and bypass an airlock by meddling with its internal wiring.
+  # Airlock Upgrades
+  It is not uncommon for plucky individuals to try and bypass an airlock by meddling with its internal wiring.
 
-Fortunately, certain countermeasures can installed into airlocks to inconvenience any would be trespassers.
+  Fortunately, certain countermeasures can installed into airlocks to inconvenience any would-be trespassers.
 
-## Medium security airlocks
-The most basic form of intrusion deterrence is to install internal steel plating that will prevent access to internal wiring of the airlock. 
+  ## Medium security airlocks
+  The most basic form of intrusion deterrence is to install a steel plating that will prevent access to internal wiring of the airlock.
```

Comment:
```suggestion
  The most basic form of intrusion deterrence is to install steel plating that will prevent access to internal wiring of the airlock.
```

Reply:
i'm not sure about this one
just steel plating doesn't ring quite right in my head

Reply:
aha, i think i got something similar (check commit [ded1f45](https://github.com/space-wizards/space-station-14/pull/26851/commits/ded1f45cc52a95e3edbc9ca0424d30291ab4ebf6))